tests/regression/test_aw.py

```diff
@@ -90,7 +90,7 @@ def epsilon(u):
                   "snes_linesearch_type": "basic",
                   "snes_monitor": None,
                   "mat_type": "aij",
-                  "snes_max_it": 10,
+                  "snes_max_it": 15,
```

Comment:
I think we wanted to do this only in complex mode:
```
from firedrake import utils

...
"snes_max_it": 15 if utils.complex_mode else 10
```

Reply:
Since I was seeing the error in both modes David suggested I apply it to both.

Reply:
Ah OK.